iteratee/src/main/scala/io/finch/iteratee/package.scala

```diff
@@ -0,0 +1,76 @@
+package io.finch
+
+import com.twitter.finagle.http.Response
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.internal._
+import io.finch.items.RequestItem
+import io.iteratee.Enumerator
+import io.iteratee.twitter.FutureModule
+import shapeless.Witness
+
+/**
+  * Iteratee module
+  */
+package object iteratee extends IterateeInstances {
+
+  import syntax._
+
+  private[finch] def enumeratorFromReader(reader: Reader): Enumerator[Future, Buf] = {

```

Comment:
Can we embed this under the defined `enumeratorBody` endpoint? I don't think it's used anywhere else.